### ISSUE TITLE
Explore: Align Query Editor Row UX

### ISSUE THREAD

**Issue by torkelo** [grafana/grafana#14987]
![2019-01-22_10-54-14_screenshot](https://user-images.githubusercontent.com/10999/51527104-1f134500-1e34-11e9-97fd-1f231e8c3bfd.png)

![2019-01-22_10-55-20_screenshot](https://user-images.githubusercontent.com/10999/51527226-541f9780-1e34-11e9-9eff-04aec35e4d74.png)

Would like to either use the same QueryRow component and design we use for dashboard panels (with some icons hidden), or find some other way to align the UX more (use same icons that mean the same thing). 

Tricky thing would be where to put add query action. I am not a big fan of the Graph & Table headers (and collapsed state & caret action). So maybe an action / toolbar under the queries with add query, and toggles for Graph & Table? 

ideas, thoughts @davkal @Ijin08 ?

**Comment by davkal** [comment]
This needs some more thought. Some pointers:

- I like to keep the motivation alive for Explore: minimal tooling, this means I'd be happy to drop up/down arrows, duplicate, pencil for Explore. The default view for text field queries should be one row per query.
- Graphical URL builders have different area requirements to text query fields. 
- What does a Clear action mean for a graphical builder?
- Panels have the result type in the query component, while Explore determines it by which panels are open. Perhaps it needs to be in the query component in Explore as well and somehow be accommodated near the text field.
- The design should be able to accommodate a text/graphical switching of the same query (e.g., when we build a graphical builder for Prometheus).

**Comment by torkelo** [comment]
> I like to keep the motivation alive for Explore: minimal tooling, this means I'd be happy to drop up/down arrows, duplicate, pencil for Explore. The default view for text field queries should be one row per query.

absolutely. Maybe as a start just align the styles the buttons & icons? So they use gf-form-label heights and margins.
